GuardianApp/AppDelegate.swift

```diff
@@ -68,6 +65,25 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
         }
     }
 
+    func application(_ application: UIApplication, handleActionWithIdentifier identifier: String?, forRemoteNotification userInfo: [AnyHashable : Any], withResponseInfo responseInfo: [AnyHashable : Any], completionHandler: @escaping () -> Void) {
```

Comment:
is this the right method? I handle the action but the app opens anyway. Shouldn't this be handled in the background? or should I call the completionHandler faster?